node/src/BaseClient.ts

```diff
@@ -2981,6 +2982,41 @@ export class BaseClient {
         return this.createWritePromise(createZRemRangeByRank(key, start, end));
     }
 
+    /**
+     * Removes all elements in the sorted set stored at `key` with lexicographical order between `minLex` and `maxLex`.
+     * See https://valkey.io/commands/zremrangebylex/ for more details.
+     *
+     * @param key - The key of the sorted set.
+     * @param minLex - The minimum lex to count from. Can be positive/negative infinity, or specific lex and inclusivity.
+     * @param maxLex - The maximum lex to count up to. Can be positive/negative infinity, or specific lex and inclusivity.
+     * @returns the number of members removed.
+     * If `key` does not exist, it is treated as an empty sorted set, and the command returns 0.
+     * If `minLex` is greater than `maxLex`, 0 is returned.
+     *
+     * @example
+     * ```typescript
+     * // Example usage of zremRangeByLex method to remove members from a sorted set based on lexicographical order range
```

Comment:
nit: I know some of our examples have this comment at the beginning of the example but I wonder if it is necessary, to me it seems redundant/self-explanatory. No worries if you prefer to leave it in though